apps/form-builder/src/app/api/forms/[formId]/formQuestions/route.ts

```diff
@@ -17,3 +28,24 @@ export async function GET(
     return NextResponse.json({ message: 'Invalid formId' }, { status: 400 });
   }
 }
+
+export async function POST(
+  request: NextRequest,
+  { params }: { params: { formId: string } }
+) {
+  const formId = params.formId;
+  await dbConnect();
+  const requestValue = await request.json();
+
+  if (!z.array(zCreateFormQuestionRequest).safeParse(requestValue)) {
+    return NextResponse.json({}, { status: 400 });
+  }
+
+  if (!zObjectId.safeParse(formId)) {
+    return NextResponse.json({}, { status: 404 });
```

Comment:
Also change this to a 400, as it indicates that the user messed up something with their request. Per the story: `on form not found (createFormQuestions returns null): 404`